application/world/services/auto_bible_generator.py

```
@@ -261,17 +261,88 @@ async def generate_and_save(
+            keywords = ["1990年深圳", "华强北 倒卖"] # 如果大模型不听话硬回 JSON，强行写死兜底，避免查出一堆乱码
+        else:
+            keywords = [k.strip() for k in content.split(',') if k.strip()]
+            
+        if not keywords:
+            keywords = [premise[:10]] # 兜底
+            
+        # 2. 执行真实搜索
+        raw_materials = []
+        for kw in keywords[:2]:
+            res = WebSearchTool.search(kw, max_results=3)
+            raw_materials.append(f"🔍 关键词【{kw}】搜索结果：\n{res}")
```

Ruff (0.15.10)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)